optimus/test_optimus.ipynb

```diff
@@ -404,13 +402,58 @@
     ")"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": 19,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "{'results': [{'id': 'ab84a3e4-ecab-44ce-98fc-d953d9d5db7c',\n",
+       "   'name': 'Optimus',\n",
+       "   'start': '2018-01-03T19:30:01.301Z',\n",
+       "   'status': 'Running'}]}"
+      ]
+     },
+     "execution_count": 19,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "cromwell_server.query(status=['Running'], names=['Optimus']).json()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "metadata": {},
+   "outputs": [
+    {
+     "ename": "NameError",
```

Comment:
Does it make sense that we're checking in a version of the notebook with an error in it? I confess I don't really know what I'm looking at here. Is there a way to get rid of this error?

Reply:
Sorry, ignore the notebook; it will be removed before I merge. I didn't realize I had checked that in.